.mailmap

```diff
@@ -1306,6 +1306,7 @@ symbolique <symbolique@users.noreply.github.com>
 tborisova <ts.borisova3@gmail.com> <ts_borisova@abv.bg>
 tnzl <you@example.com>
 tools4origins <tools4origins@gmail.com>
+user202729 <25191436+user202729@users.noreply.github.com>
```

Comment:
Is this the name that you want to be recorded in the AUTHORS file for your contributions?

Reply:
Yes.